tools/build.py

```diff
@@ -293,11 +299,10 @@ def build_cmake_args(options, for_jerry=False):
 
     # external include dir
     include_dirs = []
-    if options.target_os == 'nuttx':
-        include_dirs.append('%s/include' % options.nuttx_home)
+    if options.target_os == 'nuttx' and options.sysroot:
```

Comment:
Just noticed: at this point the `options.sysroot` will always have a path value the `adjust_options` will do an error if there is any problem. So we can remove the check from here.